dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/PI.mo

```diff
@@ -0,0 +1,100 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suit of time domain simulation tools for power systems.
+*/
+block PI "Dynawo usable version of the proportional integrator"
```

Comment:
One line before, no line after. 
Also, you can write in the comment that the input here is a discrete. No need to talk about "Dynawo usable version", it is not linked to the library in itself.

Reply:
Done